apps/invidious/config.json

```diff
@@ -19,6 +19,29 @@
       "max": 20,
       "label": "HMAC key",
       "env_variable": "INVIDIOUS_HMAC_KEY"
+    },
+    {
+      "type": "text",
+      "label": "Domain",
+      "hint": "Your local domain name for you instance",
+      "placeholder": "example.com",
+      "required": false,
+      "env_variable": "INVIDIOUS_DOMAIN"
```

Comment:
Can't we use `APP_DOMAIN` already available instead of asking the user for this extra optional field?